web/src/main/java/org/cbioportal/web/ClinicalTrialsController.java

```diff
@@ -0,0 +1,175 @@
+package org.cbioportal.web;
+
+import io.swagger.annotations.Api;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
+import org.apache.http.conn.ssl.SSLContexts;
+import org.apache.http.conn.ssl.TrustStrategy;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.cbioportal.web.config.annotation.PublicApi;
+import org.json.simple.JSONArray;
```

Comment:
Simple JSON is buggy, please switch to Jackson (https://github.com/FasterXML/jackson).